api/views/healthCheck/healthCheck.py

```diff
@@ -398,8 +398,8 @@ def check_cronjobs():
     try:
         with open('{}/cronjob.json'.format(ac.d_temp), 'r') as f:
             file_content = json.load(f)
-    except Exception:
-        return make_response(jsonify({'error': 'Data about cronjobs are not available.'}), 400)
+    except (FileNotFoundError, json.decoder.JSONDecodeError):
+        return make_response(jsonify({'error': 'Data about cronjobs are not available.', 'data': {}}), 200)
```

Comment:
Returning an `error` field along with a 200 code doesn't seem right. Maybe we could return a 500 code instead?

Reply:
Or if we consider no data to be a valid response, the error message can be removed entirely.

Reply:
Yes, you are right - the `error` message in this case is not as important as empty dict in `data` property to correctly render Healthcheck tab in Admin UI